base/standard/src/main/java/org/openscience/cdk/stereo/StereoElementFactory.java

```diff
@@ -365,6 +427,26 @@ private boolean isInSmallRing(IBond bond) {
      */
     abstract ExtendedTetrahedral createExtendedTetrahedral(int v, Stereocenters stereocenters);
 
+    /**
+     * Create an extended cis/trans bond (cumulated) given one end (see diagram
+     * below). The stereo element geometry will only be created if there is an
+     * odd number of cumulated double bonds.
+     *
+     * <pre>
+     *  C               C
+     *   \             /
+     *    C = C = C = C
+     *      ^   ^   ^
+     *      ^---^---^----- bonds
+     * </pre>
+     *
+     * @param bonds cumulated double bonds
+     * @param centers discovered stereocentres
+     * @return the extended cis/trans geometry if one could be created
+     */
+    abstract ExtendedCisTrans createExtendedCisTrans(List<IBond> bonds,
```

Comment:
Maybe a silly question, but does the order of the IBond's in the list matter?

Reply:
Yes will update the doc there... but is a package private method.